builtin/logical/pki/path_tidy.go

```diff
@@ -1089,6 +1182,65 @@ func (b *backend) doTidyExpiredIssuers(ctx context.Context, req *logical.Request
 	return nil
 }
 
+func (b *backend) doTidyAcmeAccounts(ctx context.Context, req *logical.Request, logger hclog.Logger, config *tidyConfig) error {
+	b.acmeAccountLock.Lock()
+	defer b.acmeAccountLock.Unlock()
+
+	sc := b.makeStorageContext(ctx, req.Storage)
+	list, err := sc.Storage.List(ctx, acmeThumbprintPrefix)
+	if err != nil {
+		return err
+	}
+
+	b.acmeAccountCount.Store(uint32(len(list)))
+
+	a := b.acmeState
+
+	baseUrl, err := getAcmeBaseUrl(sc, req.Path)
+	if err != nil {
+		return err
+	}
+
+	acmeCtx := &acmeContext{
+		baseUrl: baseUrl,
+		sc:      sc,
+	}
+
+	revokeCount := 0
+	deleteCount := 0
+
+	for _, thumbprint := range list {
+		revoked, deleted, err := a.TidyAcmeAccount(acmeCtx, thumbprint, time.Now().Add(config.DeleteAcmeAccountsSafetyBuffer), time.Now().Add(config.RevokeAcmeAccountsSafetyBuffer))
```

Comment:
It's early, without coffee kicking in yet, but should we not be either subtracting the safety buffers here, or using After within `TidyAcmeAccount` and not the value <account time>.Before(<time.Now()+safetybuffer>)

Reply:
Adding does work too - I've updated that on the new PR